src/software/sensor_fusion/sensor_fusion.h

```diff
@@ -82,7 +82,8 @@ class SensorFusion
      *
      * @return Ball if filtered from ball detections
      */
-    std::optional<Ball> createBall(const std::vector<BallDetection> &ball_detections);
+    std::optional<Ball> createBallProto(
+        const std::vector<BallDetection> &ball_detections);
```

Comment:
`Ball` [isn't a proto type,](https://github.com/UBC-Thunderbots/Software/blob/master/src/software/world/ball.h) this should remain as `createBall`